### DISCUSSION TITLE
Installation issue on multi mc

### DISCUSSION THREAD

**Discussion by carrotee123** [BTW-Community/BTW-Public#379] [Support] [answered]
hey so despite it being regarded as "easy" to install on multimc ive been having nothing but trouble getting it to work i make a new instance and add it to the jar and it just hangs never boots up whatsoever no longer how long i wait am i doing something wrong?? ive tried alot of different tutorials is there one definite method im missing?

**Comment by SimonMeskens** [comment, marked as answer]
https://github.com/BTW-Community/BTW-Public/wiki/Installation-(MultiMC)

**Reply by carrotee123** [reply]
thank you so much!!
